tests/test_deliver.py

```diff
@@ -628,16 +628,18 @@ def test_fetch_uppnexid(self, dbmock):
             uppnexid="this-is-the-uppnexid",
             **SAMPLECFG['deliver'])
         self.assertEquals(deliverer.uppnexid, "this-is-the-uppnexid")
-        self.assertFalse(dbmock.called,
-                         "the database should not have been queried")
+        #called once due to projectname
+        dbmock.assert_called_once_with(self.projectid)
         # if an uppnexid is not supplied in the config, the database should be consulted
+        prior = dbmock.call_count
```

Comment:
you can also reset the mock: https://docs.python.org/3/library/unittest.mock.html#the-mock-class